examples/dns/main.tf

```diff
@@ -24,4 +57,5 @@ module "dns_basic" {
 resource "humanitec_resource_definition_criteria" "dns_basic" {
   resource_definition_id = module.dns_basic.id
   app_id                 = humanitec_application.example.id
+  force_delete           = true
```

Comment:
The criteria always need a force_delete, otherwise you can't destroy the stack as Humanitec waits for the app to be gone, but TF waits for the criteria to be deleted 😅